charts/elasticsearch-cluster/templates/job-setup.yaml

```diff
@@ -115,8 +115,8 @@ spec:
 
                   RESPONSE=`eval $CURL_CMD`
 
-                  # Elasticsearch will answer at least {"acknowledged":true}, at most {"acknowledged":true, ...}
-                  if echo "$RESPONSE" | grep -v '"acknowledged":true'; then
```

Comment:
so there is no response check if its `_security/` ?

Reply:
Yes, it's because security api doesn't send response like `"acknowledged":true`, and the success response depends on the endpoint : we can't determine if the api call has worked with the response